CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 0.8.9-pre
+
+  - watch mode (-w)
+    + node sorting with custom presets/user-inserted RegEx sorts
```

Comment:
+ (s) node sorting...

ie. do the same for other lines, let's use this as a training opportunity for the reader!